javascript/javascript1/week2/script_2.js

```diff
@@ -0,0 +1,82 @@
+//*EVENT APPLICATION
+
+let todayDay = new Date();
+
+let getEventWeekday = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+
+function dateEvent(eventDay) {
+  let eventDate = new Date(todayDay);
+  eventDate.setDate(todayDay.getDate() + eventDay);
+
+  let eventWeekday = eventDate.getDay();
+
+  return getEventWeekday[eventWeekday];
+}
+
+console.log(dateEvent(5));
+
+//*Weather wear
+
+let temperature = 20;
+let clothesToWear;
+
+if (-20 <= temperature && temperature <= 0) {
+  clothesToWear = "Please do not go out from home and watch a movie";
+} else if (1 <= temperature && temperature <= 10) {
+  clothesToWear = "Use a winter Jacket and boots";
+} else if (11 <= temperature && temperature <= 15) {
+  clothesToWear = "Use a sweater and sunglases";
+} else {
+  clothesToWear =
+    "it is summer in Denmark, you can go to the beach, go for your sandals";
+}
```

Comment:
Here I would rely on the previous condition statement and only have 1 condition in each if/else if statement.

As Rasmus showed on the board then think of the numbers going from left to right. Lowest value to highest value.
`-20 -10  0  10  20  30`

Example:
```
if (temperature <= 0) {
   // code here
} else if (temperature <= 10) {
   // code here
} else if...
```
Keep in mind when a condition is met it won't continue so going from top to bottom you won't need to write the range within each conditional check.